scte35/state.go

```diff
@@ -26,15 +26,24 @@ package scte35
 
 import "github.com/Comcast/gots"
 
+const receivedRingLen = 10
+
+type receivedElem struct {
```

Comment:
We probably don't need this type, but it was added for faster lookups by pts time.  In practice, each descriptor probably has its own pts time, so it's extraneous info.